src/main/java/ch/psi/ord/core/RoCrateImporter.java

```diff
@@ -50,6 +72,60 @@ public void loadModel(Model model) {
     }
   }
 
+  public Map<String, String> importCrate(ValidationReport report, String scicatToken) {
+    Map<String, String> importMap = new HashMap<>();
+    for (var entity : report.getEntities()) {
+      if (entity.object() instanceof Publication publication) {
+        importPublication(importMap, publication, scicatToken);
+      } else {
+        throw new NotImplementedException("Only Publications are supported for now");
+      }
+    }
+
+    return importMap;
+  }
+
+  public void importPublication(
+      Map<String, String> importMap, Publication publication, String scicatToken) {
+    CreatePublishedDataDto dto = modelMapper.map(publication, CreatePublishedDataDto.class);
+    CountResponse count =
+        scicatClient
+            .countPublishedData(
+                String.format(
+                    publicationExistsFilter,
+                    DoiUtils.buildStandardUrl(publication.getIdentifier())),
+                scicatToken)
+            .getEntity();
+
+    if (count.getCount() > 0) {
+      throw new WebApplicationException(
+          "This Publication has already been imported", Status.CONFLICT);
+    }
+
+    if (dto.getPidArray().isEmpty()) {
```

Comment:
why is this done here and not similar to this? 
https://github.com/paulscherrerinstitute/scicat-rocrate/pull/68/files#diff-f524acbf7623d05d537c09b3246a1b3939e9be0b06d8c01ecddf257b8a901b63R83